examples/synchronous-proxy/README.md

```diff
@@ -0,0 +1,77 @@
+# Purpose
+
+This pattern is used when a non-workflow process needs to advance a workflow state
+machine from its initial state to its terminal state. It does this by adding input
+data to the workflow (via Signals) and receiving new information back from the
+workflow (when a secondary proxy workflow exits and returns a value).
+
+The only way to add information to a workflow is via a Signal.
+
+There are two ways to
+get information out of a workflow. One, the workflow has a Query handler and can respond
+to queries. However, this is limited in that Queries may not modify the state of the
+workflow itself. Two, the workflow can exit and return a result to its caller. This
+second approach is leveraged by the pattern to get information back from the primary
+workflow. This information could be used to determine branching behavior for the
+non-workflow caller.
+
+The flow of calls is outlined in the diagram below.
+
+![Flow Diagram](flow.png)
+
+# Explanation
+
+The primary use-case for this pattern is for a non-workflow process to *send and receive* data
+to and from a workflow. Note that a Temporal client may sent a signal to a workflow but the
```

Comment:
`client may sent a` I think you want `send`

Reply:
fixed, thanks!